src/common/tracing/src/init.rs

```diff
@@ -309,3 +301,35 @@ pub fn init_logging(
 
     guards
 }
+
+/// Creates a log filter that matches log entries based on specified target prefixes or severity.
+fn make_log_filter(prefix_filter: &str) -> impl Fn(&Metadata) -> bool {
+    let filters = prefix_filter
+        .split(',')
+        .map(|x| x.to_string())
+        .collect::<Vec<_>>();
+
+    move |meta| match_prefix_many(meta, &filters)
+}
+
+fn match_prefix_many(meta: &Metadata, prefixes: &[String]) -> bool {
+    if is_severe(meta) {
```

Comment:
is it typo?

Reply:
No, warning and error logs are considered severe, is there better term for it?

Reply:
ok. I think it's fine.